lochness/transfer/__init__.py

```diff
@@ -207,11 +208,13 @@ def send_data_over_sftp(Lochness, file_to_send: str):
     transport.close()
 
 
-def lochness_to_lochness_transfer(Lochness, general_only: bool = True):
```

Comment:
To summarize:
* `...transfer()` has been ramified into `...sftp()`, `...rsync()`, `...s3()` ?
* `...receive()` has just been renamed to `...sftp()` ?

Reply:
Yes, correct. We don't need extra receiving function except for `sftp`, in which there is zipping and unzipping involved.